src/dataaccess/mcodev0.1-datasource/EntryMapper.js

```diff
@@ -0,0 +1,900 @@
+    const newRelevantTime = new RelevantTime();
+
+    if (relevantTime.value instanceof TimePeriodV01) {
+        newRelevantTime.value = mapTimePeriod(relevantTime.value);
+    } else {
+        newRelevantTime.value = relevantTime.value;
+    }
+
+    return newRelevantTime;
+};
+
+const mapFindingResult = (value) => {
+    const newFindingResult = new FindingResult();
+
+    if (value instanceof QuantityV01) {
+        newFindingResult.value = mapQuantity(value);
+    } else if (value instanceof CodeableConceptV01) {
+        newFindingResult.value = mapPassThrough(value, CodeableConcept);
+    } else {
+        newFindingResult.value = value;
+    }
+
+    return newFindingResult;
+};
+
+const mapFindingStatus = (findingStatus) => {
+    if (findingStatus instanceof FindingStatusV01) return mapPassThrough(findingStatus, FindingStatus);
+    const newFindingStatus = new FindingStatus();
+    newFindingStatus.value = mapPassThrough(findingStatus, CodeableConcept);
+    return newFindingStatus;
+};
+
+const mapSubstanceCategory = (substanceCategory) => {
+    const newSubstanceCategory = new SubstanceCategory();
+
+    newSubstanceCategory.value = mapCodingToCodeableConcept(substanceCategory.value);
+
+    return newSubstanceCategory;
+};
+
+const mapClinicalStatus = (clinicalStatus) => {
+    const newClinicalStatus = new ClinicalStatus();
+
+    newClinicalStatus.value = mapCodingToCodeableConcept(clinicalStatus.value);
+
+    return newClinicalStatus;
+};
+
+const mapStatus = (status) => {
+    const newStatus = new Status();
+
+    newStatus.value = mapCodingToCodeableConcept(status.value);
+
+    return newStatus;
+};
+
+const mapRequestIntent = (requestIntent) => {
+    const newRequestIntent = new RequestIntent();
+
+    newRequestIntent.value = mapCodingToCodeableConcept(requestIntent.value);
+
+    return newRequestIntent;
+}
+
+const mapAdverseReaction = (adverseReaction) => {
+    const newAllergyIntoleranceReaction = new AllergyIntoleranceReaction();
+
+    newAllergyIntoleranceReaction.beginDateTime = new BeginDateTime();
+    newAllergyIntoleranceReaction.beginDateTime.value = adverseReaction.occurrenceTime.value;
+    newAllergyIntoleranceReaction.severity = mapPassThrough(adverseReaction.severity, Severity);
+    newAllergyIntoleranceReaction.manifestation = [mapPassThrough(adverseReaction.manifestation, Manifestation)];
+
+    return newAllergyIntoleranceReaction;
+};
+
+const mapCancerDisorder = (entry, klass) => {
+    const newCondition = new klass();
+
+    mapEntryInfo(entry.entryInfo, newCondition);
+    newCondition.patient = mapReference(entry._condition.patient);
+    newCondition.anatomicalLocation = entry._condition.anatomicalLocation.map(a => mapAnatomicalLocation(a));
+    newCondition.category = mapPassThrough(entry._condition.category, Category);
+    newCondition.clinicalStatus = mapClinicalStatus(entry._condition.clinicalStatus);
+    newCondition.findingResult = new FindingResult();
```

Comment:
I think this might not be the right field to use here.
`ExistenceAssertion` (and so its child classes `ConditionPresentAssertion`, `CancerDisorderPresent`, etc) has a field `findingResult` which should be a `shr.base.PresentOrAbsent` (restricted to either "Present (qualifier value)" or "Absent (qualifier value)"), and a `findingTopicCode` which should be a `shr.base.ConditionOrDiagnosisCode`
It looks like the fixtures and UI use the `findingResult` field when they should use `findingTopicCode` for the actual code of the condition which is in `condition.value` in the old model.

Reply:
Talked to Dylan and pushed up a fix for this.